app/routes/product-details.$productSlug/route.tsx

```diff
@@ -204,3 +210,49 @@ export function ErrorBoundary() {
         />
     );
 }
+
+export const meta: MetaFunction<typeof loader> = ({ data }) => {
+    const title = data?.product.name ?? 'Product Details';
+    const description = data?.product.description ?? 'Product Description';
```

Comment:
It's better to omit it entirely than to add "Product Description" as the description.